docs/source/usage_examples.rst

```diff
@@ -35,19 +35,19 @@ Using ``pych`` this function can be mapped to Chapel by changing:
 
 .. literalinclude:: /examples/test_finance_python_numpy.py
    :language: python
-   :lines: 58-67
+   :lines: 62-71
 
 to:
 
 .. literalinclude:: /examples/test_finance_chapel_numpy.py
    :language: python
-   :lines: 60-72
+   :lines: 63-75
```

Comment:
Leaving myself a reminder: this section is no longer accurate because we now require numpy arrays be flattened before sending them to Chapel programs.